[2024-08-21T06:08:36.744+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-21T06:08:37.243+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_22024-08-21.fetch_the_api scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-08-21T06:08:37.308+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_22024-08-21.fetch_the_api scheduled__2024-06-04T00:00:00+00:00 [queued]>
[2024-08-21T06:08:37.314+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-21T06:08:37.587+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): fetch_the_api> on 2024-06-04 00:00:00+00:00
[2024-08-21T06:08:37.843+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1380) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-21T06:08:37.863+0000] {standard_task_runner.py:63} INFO - Started process 1396 to run task
[2024-08-21T06:08:37.882+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_22024-08-21', 'fetch_the_api', 'scheduled__2024-06-04T00:00:00+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/final_etl.py', '--cfg-path', '/tmp/tmpnsdid8bm']
[2024-08-21T06:08:37.897+0000] {standard_task_runner.py:91} INFO - Job 229: Subtask fetch_the_api
[2024-08-21T06:08:38.830+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_22024-08-21.fetch_the_api scheduled__2024-06-04T00:00:00+00:00 [running]> on host 7e1333882d71
[2024-08-21T06:08:39.677+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_22024-08-21' AIRFLOW_CTX_TASK_ID='fetch_the_api' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-04T00:00:00+00:00'
[2024-08-21T06:08:39.710+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-21T06:08:39.931+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-21T06:08:39.940+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/data/config.py']
[2024-08-21T06:08:40.002+0000] {subprocess.py:86} INFO - Output:
[2024-08-21T06:08:40.357+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-21T06:08:40.393+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-21T06:08:40.573+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_22024-08-21, task_id=fetch_the_api, run_id=scheduled__2024-06-04T00:00:00+00:00, execution_date=20240604T000000, start_date=20240821T060837, end_date=20240821T060840
[2024-08-21T06:08:40.706+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-21T06:08:40.787+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
